h2/src/main/org/h2/mvstore/MVStoreTool.java

```diff
@@ -703,11 +700,21 @@ public static long rollback(String fileName, long targetVersion, Writer writer)
         return newestVersion;
     }
 
+    static MVMap.Builder<Object, Object> getGenericMapBuilder() {
+        return new MVMap.Builder<>().
+                keyType(GenericDataType.INSTANCE).
+                valueType(GenericDataType.INSTANCE);
+    }
+
     /**
      * A data type that can read any data that is persisted, and converts it to
      * a byte array.
      */
-    static class GenericDataType implements DataType {
+    static class GenericDataType implements DataType
+    {
+        public static GenericDataType INSTANCE = new GenericDataType();
```

Comment:
`INSTANCE` should have default visibility because `GenericDataType` isn't public, now it looks like it can be accessed from other packages, but it can't.